validator/storage_bucket_policy_only.rego

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package templates.gcp.GCPStorageBucketPolicyOnlyConstraintV1
+
+import data.validator.gcp.lib as lib
+
+deny[{
+	"msg": message,
+	"details": metadata,
+}] {
+	constraint := input.constraint
+	asset := input.asset
+	asset.asset_type == "storage.googleapis.com/Bucket"
+
+	bucket := asset.resource.data
+	enabled := bucket_policy_enabled(bucket)
```

Comment:
`bucket_policy_enabled(bucket) == false` instead?

Reply:
I have shortened this into ` bucket_policy_enabled(bucket) != true` but in reality CAI output does not display false. I've been only able to produce `"enabled": true` or `"enabled": null` so far.